sale_commission_product/views/product_view.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="product_product_form_view_commission_product" model="ir.ui.view">
```

Comment:
The commission view only shows on product variant form, but not on the main product template form

**Go to Sales > Products > Products**
The commissions section does not appear in the Product form
![Screenshot from 2020-11-10 11-25-29](https://user-images.githubusercontent.com/25710115/98701610-76571200-2347-11eb-8d64-d2774ec3574f.png)


**Go to Sales > Products > Product Variants**
The commissions section is in the view now.
![Screenshot from 2020-11-10 11-22-20](https://user-images.githubusercontent.com/25710115/98701545-65a69c00-2347-11eb-93cb-0e02b94dbe6a.png)


**Recommend to duplicate the view and inherit `product.product_template_only_form_view`**

Reply:
Hi @brian10048 I think we could consider only product.product model because, this is the model linked in the sale order lines.
And the template cloud have several variants (product.product) so i think che commission i better to stay on product.product.

have you other use case to consider? would you add a pull request ove my ?
thank you